irispy/sji.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import sunpy.cm as cm
+import sunpy.instr.iris
+import sunpy.map
+import numpy as np
+import matplotlib.colors as colors
+from astropy.io import fits
+'''
+This module provides movie tools for level 2 IRIS SJI fits file
+'''
+
+
+def sji_fits_to_cube(filelist, start=0, stop=None, skip=None, grid=False):
+    """
+    Read SJI files and return a MapCube. Inherits
+    sunpy.instr.iris.SJI_to_cube to stitch multiple sji fits
+    files stitched into one mapcube.
+    Also sets colormap and normalization
+    Assumes hdu index of 0.
+
+
+    Parameters
+    ----------
+    filelist: string or list
+        File to read, if single file string detected, will
+        create a list of length 1.
+
+    start: step
+        Temporal axis index to create MapCube from
+
+    stop: step
+        Temporal index to stop MapCube at
+
+    skip: step
+        Temporal index to skip over MapCube at
+
+
+    Returns
+    -------
+    iris_cube: sunpy.map.MapCube
+        A map cube of the SJI sequence
+    """
+
+    if type(filelist) == str:
+        filelist = [filelist]
+    iris_cube = sunpy.map.MapCube()
+    counter = 1
+    setup = sunpy.instr.iris.SJI_to_cube(filelist[0])
+    xcen = setup[0].meta.get('xcen')
+    ycen = setup[0].meta.get('ycen')
+    time = datetime.datetime.strptime(setup[0].meta.get('startobs'),
+                                      '%Y-%m-%d' + 'T' + '%H:%M:%S.%f')
+    print(xcen, ycen, time)
```

Comment:
Generally we try and minimise the use of print statements in functions. If you need to inform users of something important it is usual to raise a warning.